src/Avalonia.Controls/Primitives/Track.cs

```diff
@@ -422,6 +431,9 @@ private void ButtonChanged(AvaloniaPropertyChangedEventArgs e)
 
         private void ThumbDragged(object sender, VectorEventArgs e)
```

Comment:
Why isn't this a virtual method instead of the IsThumbDragHandledProperty?

Reply:
*D'oh* you're right! ughh im dumb 😂

Reply:
ah actually... it can't be just a virtual method @Gillibald , i dont want to subclass Track just to disable this method....